ClientGUI/ClientGUI.csproj

```diff
@@ -22,4 +22,9 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\LiteralTranslationGenerator\LiteralTranslationGenerator.csproj"
+		  OutputItemType="Analyzer"
+		  ReferenceOutputAssembly="false" />
+	</ItemGroup>
```

Comment:
Tabs mixed with spaces, also inconsistency between entries in different projects.